crates/oci-distribution/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "oci-distribution"
+version = "0.1.0"
+authors = [
+    "Matt Butcher <matt.butcher@microsoft.com>",
+    "Matthew Fisher <matt.fisher@microsoft.com>",
+    "Radu Matei <radu.matei@microsoft.com>",
+    "Taylor Thomas <taylor.thomas@microsoft.com>",
+    "Brian Ketelsen <Brian.Ketelsen@microsoft.com>",
+    "Brian Hardock <Brian.Hardock@microsoft.com>",
+    "Ryan Levick <rylevick@microsoft.com>",
+]
+edition = "2018"
+
+[dependencies]
+reqwest = { version = "0.10", features = ["json"] }
```

Comment:
Is there a reason we're not specifying minor versions? I believe the latest is 0.10.4

Reply:
I've been copying what the docs say, figuring they were right. I will start using full versions if that is better.

Reply:
I'm more confused, now. I went to look at the other Cargo.toml files in this project, and most of them do major, minor for at least some. How do we know when to include the patch version and when not to?